src/components/User/Tokens/PreissuanceRequests/components/PreissuanceRequestList.vue

```diff
@@ -226,12 +235,13 @@ export default {
     },
 
     getRequests () {
+      this.isLoading = true
       return api.requests.getPreissuanceRequests(this.asset)
         .then(r => {
           this.requests = []
           this.requests = r.records
           this.pages = r
-          this.requestsLoaded = true
+          this.isLoading = false
```

Comment:
transform to async-await, make sure that in case of fail we do not freeze screen in "Loading" state